src/main/resources/swagger/openapi.yaml

```diff
@@ -412,7 +412,7 @@ paths:
           required: true
           in: path
           description: The output type of the descriptor. Examples of allowable values are
-            "CWL", "WDL", and "NextFlow."
+            "CWL", "WDL", and "NFL".
```

Comment:
This was from a comment by Paolo
https://github.com/dockstore/dockstore/pull/1101#issuecomment-357516486

Reply:
Right.  But I would rather use the `UPPERCASE` of their file extension "`.nf`" ==> `NF` if one exists. This won't work for Galaxy's Toolshed XML since they just use the `.xml` as the file extension though.

Reply:
I was proposing to keep consistent with the `CWL`, `WDL` scheme. `NEXTFLOW` also makes sense to me.

Reply:
I'm ok either way, is there a strong preference on this one?